### DISCUSSION TITLE
Nuxt bridge set page title

### DISCUSSION THREAD

**Discussion by Pflyg** [nuxt/nuxt#18006] [Questions]
I am trying to build a page using Nuxt, but I just cannot figure out how to set the page title.
This is what I currently have:

```html
<template>
  <div>
    <Test/>
    </div>
</template>

<script lang="ts">
import Vue from 'vue'
import Test from '~/components/Test.vue'

export default Vue.extend({
    name: "IndexPage",
    meta: {
      title: "test"
    },
    components: { Test },
})
</script>
```

The title is however still using the one defined in nuxt.config

EDIT: using useMeta doesn't do anything either
```html
<script setup>
  useMeta({ title: 'My page' })
</script>
```

**Comment by psycongaroo** [comment]
Hi, try with:

`import { useNuxt2Meta } from '#app'
  useNuxt2Meta({
    title: 'My Nuxt App',
  })`

or for useMeta you should enable it in nuxt.config.ts:

`bridge: {
    meta: true
}`
